aws-ecsfargate-terraform/README.md

```diff
@@ -1,72 +1,117 @@
 
 ## Prerequisites
 
 Before beginning, familiarize yourself with [PREPARATION.md](/PREPARATION.md) and complete the necessary steps there.
 
-- the [terraform](https://learn.hashicorp.com/tutorials/terraform/install-cli) command line tools
+- [Terraform](https://www.terraform.io/downloads)
 - `scimsession` file and bearer token (as seen in `PREPARATION.md`)
-- (Optional) DNS Zone in Route53
 
-## Log In With `aws`
+## Sign in with `aws`
 
 Ensure you are authenticated with the `aws` tool in your local environment.
 
 See [Terraform AWS Authentication](https://registry.terraform.io/providers/hashicorp/aws/latest/docs#authentication) for more details.
 
 ## Configuration
 
-### Copy Configuration
+### Copy configuration template
 
 Copy `terraform.tfvars.template` to `terraform.tfvars`:
 
 ```bash
 cp terraform.tfvars.template terraform.tfvars
 ```
 
-### Copy `scimsession` File
+### Copy `scimsession` file
 
-Copy the `scimsession` file in the terraform code directory:
+Copy the `scimsession` file in the Terraform code directory:
 
 ```bash
-cp /path/to/scimsession ./
+cp <path>/scimsession ./
 ```
 
-This will install the `scimsession` file automatically in your instance.
+This will automatically create an AWS secret containing the contents of the `scimsession` file in your instance.
 
-NOTE: If you skip this step or the installation of the `scimsession` file is not successful, you can perform this step manually afterwards. Ensure you `base64` encode the `scimsession` file, and store it in a secret as plain text (not in json, and not wrapped in quotation marks).
+**Note:** If you skip this step or the installation of the scimsession file is not successful, you can create the required AWS secret manually. Ensure you `base64` encode the `scimsession` file, and store it in a secret as **plain text** (not as JSON, and not wrapped in quotation marks):
 
 ```bash
 # only required if the automatic installation of the 'scimession' file is not successful
-cat /path/to/scimsession | base64
+cat <path>/scimsession | base64
 # copy the output to Secrets Manager
 ```
 
-### (Optional) Region
+### Region
 
-Create a region entry in `terraform.tfvars` for what region you're deploying in (default is `us-east-1`).
+Set the `aws_region` variable in `terraform.tfvars` to the AWS region you're deploying in (the default is `us-east-1`).
 
-### Domain Name
+### Domain name
 
-This example uses AWS Certificate Manager to manage the certificate. Save the full domain name you want to use as domain_name in `terraform.tfvars`:
+This example uses AWS Certificate Manager to manage the required TLS certificate. Save the full domain name you want to use as `domain_name` in `terraform.tfvars`:
 
 ```
-    domain_name = "scim-bridge.yourcompany.com"
+domain_name = "<scim.example.com>"
```

Comment:
In a similar vein, I understand the the change to `scim.example.com` but I'm not sure we should wrap it in `<>`.

Reply:
(See [above](https://github.com/1Password/scim-examples/pull/169#discussion_r788213623). 👆🏻)